scripts/README.md

```diff
@@ -0,0 +1,35 @@
+# Scripts
+
+## update_library_version.sh
+
+This script updates the version of a specified library in Pipfile and requirements-elyra.txt files, only if the new version is higher. It can optionally run `pipenv lock` after updating the version.
+
+### Usage
+
+```sh
+./update_library_version.sh <directory> <library_name> <new_version> <include_paths_with> <exclude_paths_with> <lock_files>
+```
+
+### Parameters
+
+- `directory`: The root directory to start searching for Pipfile and requirements-elyra.txt files.
+- `library_name`: The name of the library to update.
+- `new_version`: The new version to set for the library.
+- `include_paths_with`: A pipe-separated list of substrings; only files in directories containing at least one of these substrings will be processed.
+- `exclude_paths_with`: A pipe-separated list of substrings; files in directories containing any of these substrings will be excluded.
+- `lock_files`: Whether to run `pipenv lock` after updating the library version (`true` or `false`).
+
```

Comment:
```suggestion

```

I'd not put it here, it's enough to have it on top of the script, IMO. Fewer things to keep in sync. The Examples is what matters.